codebundles/aws-c7n-ebs-health/runbook.robot

```diff
@@ -0,0 +1,158 @@
+        ${ebs_volume_list}=    Create List
+    END
+
+    # Generate issues if any unused EBS volumes are in the list
+    IF    len(@{ebs_volume_list}) > 0
+        FOR    ${item}    IN    @{ebs_volume_list}
+            RW.Core.Add Issue        
+            ...    severity=2
+            ...    expected=EBS volumes in AWS Account `${AWS_ACCOUNT_NAME}` should be attached or in use
+            ...    actual=EBS volume `${item["VolumeId"]}` in AWS Account `${AWS_ACCOUNT_ID}` is unused
+            ...    title=Unused EBS volume `${item["VolumeId"]}` detected in AWS Account `${AWS_ACCOUNT_ID}`
+            ...    reproduce_hint=Review the volume details and usage in the AWS Management Console or CLI.
+            ...    details=${item}        # Include details such as volume ID, size, and region.
+            ...    next_steps="Escalate to service owner to review of unattached AWS EBS volume `${item["VolumeId"]}` in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`. \n Delete unattached AWS EBS volume `${item["VolumeId"]}` in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`."
+        END
+    END
+
+
+List Unencrypted EBS Volumes in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`
+    [Documentation]  Check for unattached EBS volumes in the specified region. 
+    [Tags]    ebs    storage    aws    volume
+    ${c7n_output}=    RW.CLI.Run Cli
+    ...    cmd=custodian run -r ${AWS_REGION} --output-dir ${OUTPUT_DIR}/aws-c7n-ebs-health ${CURDIR}/unencrypted-ebs-volumes.yaml --cache-period 0
+    ...    secret__aws_account_id=${AWS_ACCESS_KEY_ID}
+    ...    secret__aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
+    ${report_data}=     RW.CLI.Run Cli
+    ...    cmd=cat ${OUTPUT_DIR}/aws-c7n-ebs-health/unencrypted-ebs-volumes/resources.json 
+    RW.Core.Add Pre To Report    ${c7n_output.stdout} 
+
+    ${parsed_results}=    CloudCustodian.Core.Parse EBS Results
+    ...    input_dir=${OUTPUT_DIR}/aws-c7n-ebs-health
+    RW.Core.Add Pre To Report    ${parsed_results}  
+
+    ${clean_output_dir}=    RW.CLI.Run Cli    cmd=rm -rf ${OUTPUT_DIR}/aws-c7n-ebs-health/unencrypted-ebs-volumes
+   
+    TRY
+        ${ebs_volume_list}=    Evaluate    json.loads(r'''${report_data.stdout}''')    json
+        Log    ${report_data.stdout}
+    EXCEPT
+        Log    Failed to load JSON payload, defaulting to empty list.    WARN
+        ${ebs_volume_list}=    Create List
+    END
+
+    IF    len(@{ebs_volume_list}) > 0
+        FOR    ${item}    IN    @{ebs_volume_list}
+            RW.Core.Add Issue        
+            ...    severity=2
+            ...    expected=EBS volumes in AWS Account `${AWS_ACCOUNT_ID}` should be unencrypted or in use
+            ...    actual=EBS volume `${item["VolumeId"]}` in AWS Account `${AWS_ACCOUNT_ID}` is unencrypted
+            ...    title=Unencrypted EBS volume `${item["VolumeId"]}` detected in AWS Account `${AWS_ACCOUNT_ID}`
```

Comment:
Please include the AWS Region in the Title, which helps with the WHERE component of the title (A better WHAT & WHERE makes for better understanding /searching through titles. The `qualifer` in the template often helps instruct what detail should be included in the WHERE.